### ISSUE TITLE
Research Select Components with Accessibility

### ISSUE THREAD

**Issue by davezen1** [US-CBP/cbp-theme#27]
As a user of the CBP Theme, I would like examples of select boxes that are a11y (accessible) so that my application can be more accessible.

https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_01

https://make.wordpress.org/accessibility/2015/09/07/accessibility-usertest-select2/
- We currently demo select2 and selectize which result in a11y issues because of the lack of labels on the rendered input tags. We will need to either provide fixes to the libraries or create our own.

**Comment by davezen1** [comment]
@d-amit @djKianoosh After discussing with @cmterrylynn we have some leeway with regards to select2 and selectize.

**Comment by davezen1** [comment]
Closing per previous comment.
